source/core.py

```diff
@@ -130,14 +129,43 @@ def handleReplaceCLIArg(cliArgument: str) -> bool:
 		if updateCheck and not config.conf['update']['askedAllowUsageStats']:
 			# a callback to save config after the usage stats question dialog has been answered.
 			def onResult(ID):
-				import wx
 				if ID in (wx.ID_YES,wx.ID_NO):
 					try:
 						config.conf.save()
 					except:
 						pass
 			# Ask the user if usage stats can be collected.
 			gui.runScriptModalDialog(gui.startupDialogs.AskAllowUsageStatsDialog(None), onResult)
+	addonFailureMessages: list[str] = []

```

Comment:
please move this block out to a helper function - this adds a lot of complexity to the function